09_Loading_Elf/01_Elf_Theory.md

```diff
@@ -98,7 +98,7 @@ This type means that we are expected to load the contents of this program header
 To load our simple, statically-linked, program the process is as follows:
 
 1) Load the ELF file in memory somewhere.
-2) Validate the ELF header by checking the machine type matches what we expect (is this an x86_64 program?).
+2) Validate the ELF header by checking the machine type matches what we expect (is this an x86_64 program?), this is usually done by parsing the `e_ident` member.
```

Comment:
correct, but can drop the 'usually'. We know this is contained in the e_ident field, as per the spec. There is no other behaviour in a valid elf file.